spec/models/user_spec.rb

```diff
@@ -0,0 +1,51 @@
+require 'rails_helper'
```

Comment:
- [ ] It would be better if we fix all our failing test cases. Let ensure all cases pass before submitting another code review
![Screenshot from 2022-01-28 14-35-42](https://user-images.githubusercontent.com/46388113/151565570-a1e25e14-0efd-4417-b443-a2f296df20d4.png)